core/src/main/resources/hudson/slaves/JNLPLauncher/help_bg.properties

```diff
@@ -20,7 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-# Allows an agent to be launched using <a href="https://en.wikipedia.org/wiki/Java_Web_Start" target="_blank">Java Web Start</a>.<br>\ 
+# Allows an agent to be launched using <a href="https://en.wikipedia.org/wiki/Java_Web_Start" target="_blank">Java Web Start</a>.<br> \
```

Comment:
I think we can remove all the \ of these comments, like this:

```suggestion
# Allows an agent to be launched using <a href="https://en.wikipedia.org/wiki/Java_Web_Start" target="_blank">Java Web Start</a>.<br>
```

Reply:
Technically they're comments, so nothing matters. But the spirit is such that I think they want to belong, and I don't want to make a sweeping change.